docs/pages/faq.rst

```diff
@@ -0,0 +1,51 @@
+Frequently Asked Question
+=========================
+
+
+Incorrect encoding
+------------------
+
+In Query protocol, Minecraft uses ISO 8859-1 for encoding all text (like MOTD,
+server name, etc.). This can cause problems with non-latin characters. To fix
+such error, you can re-encode text into UTF-8.
+
+.. code-block:: python
+
+    >>> query = JavaServer.lookup("my-server-ip.com").query()
+    >>> query.motd
+    'Ð\x9fÑ\x80Ð¸Ð²Ñ\x96Ñ\x82!'
+    >>> query.motd.encode("iso-8859-1").decode("utf-8")
+    'Привіт!'
+
+:attr:`query.motd <mcstatus.querier.QueryResponse.motd>` here can be anything,
+that contain incorrect encoding.
+
+
+How to get server image?
+------------------------
+
+Bedrock servers do not provide icons, so you can't get server image from them.
+For Java servers, you can use :attr:`status.favicon <mcstatus.pinger.PingResponse.favicon>`
+attribute. It will return `Base64 <https://en.wikipedia.org/wiki/Base64>`_
+encoded PNG image, so you can decode it in that way:
+
+.. code-block:: python
+
+    import base64
+
+    with open("server-favicon.png", "wb") as server_favicon_file:
+        server_favicon_file.write(base64.decodebytes(status.favicon))
+
+
+
+``RuntimeError: Event Loop is closed``
```

Comment:
Is there some mcstatus issue regarding this happening? What exactly causes this?

Reply:
Oh, that was tracked by #337, and is fixed now.

Reply:
Removed in [last force-push](https://github.com/py-mine/mcstatus/compare/9086ff3d9484d8cde97d4439741329a1a487017d..f71fea6b2261db29714c612dcf737c7c75465482), thanks!